src/components/App/index.tsx

```diff
@@ -36,23 +37,45 @@ const ProjectsPage = React.lazy(() =>
 type ModiteListTypes = 'globe' | 'list'
 
 const Inner = () => {
+//  const listOptions = JSON.parse(localStorage.getItem("list-options")) || {
```

Comment:
should we keep this comment?

Reply:
(and other commented out lines in this module)

Reply:
removed commented out lines.

They were there for my reference.